systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -992,6 +1002,80 @@ void testScaleConnectAndConnectorSubresource() {
         }
     }
 
+    @Test
+    void testMountingSecretAndConfigMapAsVolumes() {
```

Comment:
You seem to use both here?

```suggestion
    void testMountingSecretAndConfigMapAsVolumesAndEnvVars() {
```

Reply:
Yes! Thanks :)

Reply:
Done